elytron/src/main/java/org/wildfly/extension/elytron/ElytronPersistentResourceXMLDescriptionFactory.java

```diff
@@ -0,0 +1,230 @@
+/*
+ * Copyright The WildFly Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.wildfly.extension.elytron;
+
+import org.jboss.as.controller.AttributeMarshallers;
+import org.jboss.as.controller.AttributeParsers;
+import org.jboss.as.controller.PathElement;
+import org.jboss.as.controller.PersistentResourceXMLDescription;
+
+import java.util.function.Function;
+
+import static org.jboss.as.controller.PersistentResourceXMLDescription.builder;
+import static org.wildfly.extension.elytron.ElytronDescriptionConstants.DIR_CONTEXTS;
+import static org.wildfly.extension.elytron.ElytronDescriptionConstants.ENCRYPTION;
+import static org.wildfly.extension.elytron.ElytronDescriptionConstants.EXPRESSION;
+import static org.wildfly.extension.elytron.ElytronDescriptionConstants.EXPRESSION_RESOLVER;
+import static org.wildfly.extension.elytron.ElytronDescriptionConstants.PERMISSION_SETS;
+import static org.wildfly.extension.elytron.ElytronDescriptionConstants.SECURITY_PROPERTY;
+import static org.wildfly.extension.elytron.PermissionMapperDefinitions.PERMISSIONS;
+
+public enum ElytronPersistentResourceXMLDescriptionFactory implements Function<ElytronSubsystemSchema, PersistentResourceXMLDescription> {
+    INSTANCE;
```

Comment:
Should this be `static final ElytronPersistentResourceXMLDescriptionFactory INSTANCE = new ElytronPersistentResourceXMLDescriptionFactory();`?

Reply:
This enum was removed